jetty-server/src/main/java/org/eclipse/jetty/server/CustomRequestLog.java

```diff
@@ -311,6 +313,15 @@ public CustomRequestLog(RequestLog.Writer writer, String formatString)
         }
     }
 
+    /**
+     * This allows you to set a custom filter to decide whether to log a request or omit it from the request log.
+     * @param filter - a BiPredicate which returns true if this request should be logged.
+     */
```

Comment:
Might need to describe that this filter is evaluated after the other filtering, such as ignore paths?